### PR TITLE
Add util helpers for explorer urls

### PR DESCRIPTION
**Pull request by junkai121**

### CODE REVIEW

**Review by veado — COMMENTED** [review]
Can we use new `utils` helpers in `client`, too (to avoid duplication of same code)?

**Comment by junkai121** [comment]
> Can we use new `utils` helpers in `client`, too (to avoid duplication of same code)?

In client side, it uses the explorer urls from constructor. Currently user can set explorer url when creating client.
BTW in utils helper, it will use the default explorer urls.

File: packages/xchain-thorchain/src/util.ts [hunk 1]
```diff
@@ -241,3 +241,58 @@ export const getDefaultFees = (): Fees => {
...
+  }
+}
+
+/**
+ * Get the explorer url.
+ *
+ * @param {Network} network
+ * @returns {string} The explorer url for thorchain based on the given network.
+ */
+export const getExplorerUrl = (network: Network): string => {
```
Comment (veado) [inline packages/xchain-thorchain/src/util.ts:263]: `getExplorerUrl` -> `getDefaultExplorerUrlByNetwork`

File: packages/xchain-thorchain/src/util.ts [hunk 2]
```diff
@@ -241,3 +241,58 @@ export const getDefaultFees = (): Fees => {
...
+}
+
+/**
+ * Get the explorer url for the given address.
+ *
+ * @param {Network} network
+ * @param {Address} address
+ * @returns {string} The explorer url for the given address.
+ */
+export const getExplorerAddressUrl = (network: Network, address: Address): string => {
```
Comment (veado) [inline packages/xchain-thorchain/src/util.ts:274]: `getExplorerAddressUrl` -> `getDefaultAddressUrl`

File: packages/xchain-thorchain/src/util.ts [hunk 3]
```diff
@@ -241,3 +241,58 @@ export const getDefaultFees = (): Fees => {
...
+}
+
+/**
+ * Get the explorer url for the given node.
+ *
+ * @param {Network} network
+ * @param {Address} node address
+ * @returns {string} The explorer url for the given node.
+ */
+export const getExplorerNodeUrl = (network: Network, address: Address): string => {
```
Comment (veado) [inline packages/xchain-thorchain/src/util.ts:285]: `getExplorerNodeUrl` -> `getDefaultExplorerNodeUrl`

**Review by veado — APPROVED** [review]

**Pull request merged**